Assets/Scripts/Model/Types/PlayerState.cs

```diff
@@ -45,5 +45,30 @@ public static PlayerState Get(int? state)
 
             return PlayerStates[key];
         }
+
+        public string getDescription()
```

Comment:
Change this to a property instead:

public string Description => {
  switch ...
}

Reply:
Was not able to do it with lambda without errors. Ended up with a regular getter instead.